mysql-test/extra/rpl_tests/rpl_gtids_restart_slave_io_lost_trx.test

```diff
@@ -10,14 +10,14 @@
 # IO thread right after queuing an event of a given type, starting the IO
 # thread again to fully retrieve the partially received transaction.
 #
-# Additionally, this test also tests partial transactions with MASTER_DELAY
-# configured and verifies that the applier thread doesn't wait for MASTER_DELAY
+# Additionally, this test also tests partial transactions with SOURCE_DELAY
```

Comment:
@kamil-holubicki The test was failing on Jenkins because it had not been fixed in 8.0.23. However, it was fixed in 8.0.24 by  https://github.com/percona/percona-server/commit/2bc2c2c17496bbc00060379c17a9d0dd5f5df72e and the same was ported to PXC-8.0.23 upfront. So, there will not be any conflicts on future.